<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationForm</class>
 <widget class="QDialog" name="RegistrationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Регистрация</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>New user registration</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel">
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="usernameEdit">
      <property name="placeholderText">
       <string>Enter username</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel">
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="passwordEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Enter password</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel">
      <property name="text">
       <string>Password confirmation:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="confirmPasswordEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Repeat password</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel">
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="emailEdit">
      <property name="placeholderText">
       <string>Enter email</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QWidget">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="registerButton">
         <property name="text">
          <string>Register</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="statusLabel">
 <property name="alignment">
  <set>Qt::AlignCenter</set>
 </property>
</widget>
 </item>
 </layout>
 </widget>
 </item>
 </layout>
</widget>
<resources/>
<connections/>
</ui>
